tools/cli/go-whisk-cli/main.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
     // Rest of CLI uses the Properties struct, so set the build time there
     commands.Properties.CLIVersion = CLI_BUILD_TIME
+    os.Setenv("CLI_Version", CLI_BUILD_TIME)
```

Comment:
avoid using "global" style variables.  instead set a value on the http config object during it's initialization